src/index.html

```diff
@@ -3,10 +3,14 @@
 <head>
     <meta charset="utf-8">
     <title>React Comments</title>
-    <script type="text/javascript" src="js/scripts.min.js"></script>
+    
     <link rel="stylesheet" href="css/styles.min.css" type="text/css" />
 </head>
 <body>
-    <h1>React Comments</h1>
+    <h1>React Commentss</h1>
+    <div id="root"></div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.12.2/react.min.js"></script>
```

Comment:
You've already specified `react` and `react-dom` as dependencies, so they will be added to a bundle. You don't have to add them one more time via CDN libs.